sources/packages/forms/src/form-definitions/studentinformation.json

```diff
@@ -654,194 +835,813 @@
           "addons": [],
           "tree": false,
           "lazyLoad": false,
-          "id": "epucz5o"
+          "id": "ebsmrdq"
         },
         {
-          "label": "Email",
-          "labelPosition": "top",
+          "title": "BCeID create mode when the user must provide all the data.",
           "labelWidth": "",
           "labelMargin": "",
-          "placeholder": "",
-          "description": "",
+          "theme": "default",
           "tooltip": "",
-          "prefix": "",
-          "suffix": "",
-          "widget": {
-            "type": "input"
-          },
-          "displayMask": "",
-          "customClass": "",
-          "tabindex": "",
-          "autocomplete": "",
+          "customClass": "formio-panel-unset",
+          "collapsible": false,
           "hidden": false,
-          "hideLabel": false,
-          "mask": false,
-          "autofocus": false,
-          "spellcheck": true,
-          "disabled": true,
-          "tableView": true,
+          "hideLabel": true,
+          "disabled": false,
           "modalEdit": false,
+          "key": "bcscReadOnlyFieldsForBCeIdAllMustBeEditableForCreation1",
```

Comment:
can we give a different key instead of `*1`

Reply:
Changed.